fdbserver/workloads/AsyncFileCorrectness.actor.cpp

```diff
@@ -300,18 +300,14 @@ struct AsyncFileCorrectnessWorkload : public AsyncFileWorkload
 			{
 				int64_t maxOffset;
 
-				//Reads should not exceed the extent of written data
-				if(info.operation == READ)
-				{
+				// Reads should not exceed the extent of written data
+				if (info.operation == READ) {
 					maxOffset = fileSize - 1;
-					if(maxOffset < 0)
-						info.operation = WRITE;
+					if (maxOffset < 0) info.operation = WRITE;
+					// Only allow reads once the file has gotten large enough (to prevent blocking on locks)
+					if (maxOffset < targetFileSize / 2) info.operation = WRITE;
```

Comment:
This fixes a potential use of uninitialized memory. Not sure why valgrind doesn't seem to catch it. It's branching on maxOffset, which could have been uninitialized in the old version